addons/web/static/src/views/fields/selection/filterable_selection_field.js

```diff
@@ -9,6 +9,7 @@ import { SelectionField, selectionField } from "@web/views/fields/selection/sele
 export class FilterableSelectionField extends SelectionField {
     static props = {
         ...SelectionField.props,
+        whitelisted_fname: { type: String, optional: true },
```

Comment:
```suggestion
        whitelist_fname: { type: String, optional: true },
```
sounds less weird to me